srcpkgs/vpnc/patches/0001-fix-sys-includes-for-musl.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/vpnc.c b/src/vpnc.c
+index ef1930a..36cae9b 100644
+--- ./src/vpnc.c
++++ ./src/vpnc.c
+@@ -24,7 +24,7 @@
+ #define _GNU_SOURCE
+ #include <assert.h>
+ #include <unistd.h>
+-#include <sys/fcntl.h>
++#include <fcntl.h>
```

Comment:
musl just prints a warning if one does this; do they have `-Werror`?

Reply:
No, it did at some point during my testing. Removing the patch and offer it upstream instead.